cluster.go

```diff
@@ -79,3 +79,17 @@ func (cl *Cluster) Tasks(ctx context.Context) (Tasks, error) {
 
 	return tasks, nil
 }
+
+func (cl *Cluster) Ceph(ctx context.Context) (*Ceph, error) {
+	ceph := &Ceph{
+		client: cl.client,
+	}
+
+	// TODO?
```

Comment:
Not sure which permissions I actually need to access that endpoint. I've copied the structure from "cluster". Was its intention to ensure always get a cluster/ceph instance out of that function but also a potential error?

Reply:
is there any reason you would want the Ceph if you are not authorized? I think I like this function as is